driver_scripts/driver_grid.dell.sh

```diff
@@ -56,8 +56,9 @@ module load EnvVars/1.0.2
 module load lsf/10.1
 module load ips/18.0.1.163
 module load impi/18.0.1
-module load NetCDF/4.5.0
 module load HDF5-serial/1.10.1
+module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
+module load netcdf_parallel/4.7.4
 module list
```

Comment:
may be this is an oversight, but this script is still loading modules explicitly. Shouldn't it be sourcing modulefiles?

Reply:
Good catch.

Reply:
Fixed.